pkg/cluster/generateconfig.go

```diff
@@ -142,6 +141,7 @@ func (m *manager) generateInstallConfig(ctx context.Context) (*installconfig.Ins
 						EncryptionAtHost: m.doc.OpenShiftCluster.Properties.MasterProfile.EncryptionAtHost == api.EncryptionAtHostEnabled,
 						OSDisk: azuretypes.OSDisk{
 							DiskEncryptionSetID: m.doc.OpenShiftCluster.Properties.MasterProfile.DiskEncryptionSetID,
+							DiskSizeGB:          1024,
```

Comment:
https://github.com/openshift/installer/pull/4666 breakes our code: we were relying on defaults set here and the fact that installer was mutating the isntallconfig object:

https://github.com/openshift/installer/blob/94125b21304a574ae5bc98039f8eb7f518293b83/pkg/asset/machines/master.go#L252-L253

But these defaults are not longer propogated to the installer config becuase this:

https://github.com/openshift/installer/blob/94125b21304a574ae5bc98039f8eb7f518293b83/pkg/asset/machines/master.go#L150

changed to this in 4.8:

https://github.com/openshift/installer/blob/c06e9b1562ccb76ac23163fb8db35f231ead6e97/pkg/asset/machines/master.go#L150

Fixed it by adding default value to our `pkg/cluster/generateconfig.go`.